error: 'ngx_table_elt_t' has no member named 'next'

When compiled brotli module into nginx-1.8.1 version, it causes an error below:
 error: 'ngx_table_elt_t' has no member named 'next'
  322 |   h->next = NULL;

tried with a older commit version of brotli earlier, that was supposed to work and yet that's not an ideal way of using the module

/kind bug

We went to upgrade alpine to 3.18.4 and I am seeing this error in the cloud build logs for 1.9.1 

````
#9 563.4 /tmp/build/ngx_brotli/static/ngx_http_brotli_static_module.c:264:25: error: 'ngx_table_elt_t' has no member named 'next'
````
https://console.cloud.google.com/cloud-build/builds;region=global/a0ac43a8-1ea9-4c46-9461-24cf91555c79?project=k8s-staging-ingress-nginx

--- a/images/nginx/rootfs/build.sh
+++ b/images/nginx/rootfs/build.sh
@@ -466,8 +466,10 @@ make install
 
 # Get Brotli source and deps
 cd "$BUILD_PATH"
-git clone --depth=1 https://github.com/google/ngx_brotli.git
+git clone --depth=100 https://github.com/google/ngx_brotli.git
 cd ngx_brotli
+# https://github.com/google/ngx_brotli/issues/156
+git reset --hard 63ca02abdcf79c9e788d2eedcc388d2335902e52
 git submodule init
 git submodule update
 

